doc/cephfs/fs-nfs-exports.rst

```diff
@@ -63,6 +63,24 @@ Show NFS Ganesha Cluster Information
 
 This displays ip and port of deployed cluster.
 
+Set Customized Ganesha Configuration
+====================================
+
+.. code:: bash
+
+    $ ceph nfs cluster config set <clusterid> -i <config_file>
+
+With this the nfs cluster will use the specified config instead of default.
```

Comment:
Uh, I think this was supposed to be a supplemental config, right? It's not replacing any default. You can mention it's loaded last (that matters to Ganesha)?

Reply:
Done.